model/visor/processing.go

```diff
@@ -164,6 +164,13 @@ func (p *ProcessingActor) ParentTipSetKey() (types.TipSetKey, error) {
 	return TipSetKeyFromString(p.ParentTipSet)
 }
 
+func (p *ProcessingActor) MarkComplete(completedAt time.Time, errorDetected error) {
+	if errorDetected != nil {
```

Comment:
I think we will want to make sure ClaimedUntil is zero. It's not strictly necessary but it keeps the data clean.